components/compliance-service/inspec-agent/resolver/resolver.go

```diff
@@ -485,17 +485,28 @@ func (r *Resolver) handleAzureVmNodes(ctx context.Context, m *manager.NodeManage
 	return r.handleManagerNodes(ctx, m, nodeCollections, job)
 }
 
-func (r *Resolver) handleInstanceCredentials(ctx context.Context, instanceCreds []*manager.CredentialsByTags, node *manager.ManagerNode) ([]*inspec.Secrets, nodeInfo) {
-	var nodeInfo nodeInfo
+func handleNodeInfo(node *manager.ManagerNode) nodeInfo {
```

Comment:
I know we use `handle` in many of the other function names, but I'm wondering if `nodeInfoFromManagerNode` would be a more clear name?